themes/default/content/blog/pulumi-service-updates/index.md

```diff
@@ -0,0 +1,106 @@
+- create new organizations
+- access your favorite stacks (read on to find out more!)
+- directly access sub-pages like Billing or Access Tokens
+- send us your feedback!
+
+![""](dashboard-nav-collapsed.png)
+
+We made these changes to help you move through the Pulumi Service without losing your place and ensure constant access to key pages. Even if you collapse the navigation pane for more screen real estate, you can still get where you need to go!
+
+If you're looking for your account details, they have moved into the black utility toolbar. To access your account settings or logout of the Pulumi Service, select your profile image at the top right of your browser window.
+
+## Stack Favorites
+
+You can now pin your favorite stacks, so you can quickly access the stacks you need most. This reduces your time spent combing through projects. Look for the star icon next to any stack name and select it to start building out your favorites list.
+
+![""](stack-navigation.png)
+
+Your favorites list is your own. Even if you're part of an organization, your favorites won't be shared with anyone other than you. And you can have as many favorites as you like!
+
+You can access the favorites list from two key locations in the Service: the primary navigation pane and the dashboard. All your favorites will appear in the navigation pane under the “Stacks” heading. You'll also have your list of favorites in a new dashboard card.
+
+![""](stacks-navigation-favorites-card.png)
+
+## Pulumi Service dashboard redesign
+
+linter fun _image will be added here_
+
+Lastly, we’ve also made significant changes to the Pulumi Service dashboard. Now you can access more information to help survey your operations at a glance.
+
+Organizations can access some quick usage statistics at the top of their dashboards:
+
+- member count and +/- change in the past week
+- total stack count and percent change in the past week
+- total resource count and percent change in the past week
+- number of successful updates in the past day, week, month, and year
+
+![""](dashboard-quick-stats.png)
+
+If you're part of an organization, you'll also see a “Members” card that tracks recently joined members. If you're an admin, you can manage any pending invites from this card.
+
+We've also exposed more operation updates in the “Latest Stack Activity” card. You can see up to ten most recent updates and favorite those stacks directly from the dashboard card.
+
+![""](members-stack-activity-cards.png)
+
+## Try out the Pulumi Service today
+
+We'd love for you to take this and improved Pulumi Service for a ride! If you haven't already, you can [create an account](https://app.pulumi.com/signup) and experience these changes for yourself. Don't hesitate to [send us feedback](https://www.pulumi.com/contact/) about any thoughts and feelings you have about these changes or any and all things Pulumi! We're more than happy to learn from our community.
```

Comment:
first sentence has grammatical issue

Reply:
should we plug the community slack as well? i thought that the contact form is typically used for sales queries.

Reply:
is there also a way to submit GH issues for the Service?